docs/faq.rst

```diff
@@ -631,6 +631,9 @@ already used.
 By default, ctime (change time) is used for the timestamps to have a rather
 safe change detection (see also the --files-cache option).
 
+Furthermore, pathnames recorded in files cache are always absolute, even if you specify
+source directories with relative pathname. If relative pathnames are stable, but absolute are
+not (for example if you mount a filesystem without stable mount points for each backup or if you are running the backup from a filesystem snapshot whose name is not stable), borg will assume that files are different and will report them as 'added', even though no new chunks will be actually recorded for them.
```

Comment:
Good point. Did you try if a bind-mount to a stable path (and then giving that stable path to borg) helps?

If it helps, it could be mentioned here.

Reply:
Yes, bind mount helps (this is the solution I ended up using). Amended